### DISCUSSION TITLE
求解惑nameserver-policy下一个键只允许对应一个DNS值吗？可以加多个吗？

### DISCUSSION THREAD

**Discussion by QQ29701** [MetaCubeX/ClashMetaForAndroid#406] [Q&A]
我试着加多个但是不知道语法是什么？是用逗号分隔吗？尝试之后不行，比如geosite:cn,private我想加119.29.29.29和223.5.5.5这两个DNS，要怎么写才生效？

**Comment by QQ29701** [comment]
忘记说，在GUI中添加，不是手搓配置文件

**Comment by Ben-md5** [comment]
+1, 同样遇到这个问题

**Comment by lumine0915** [comment]
建议换用支持更复杂覆写规则的客户端（如FlClash），个人感觉CMFA近期之内不会做什么功能上的更新了。

另外不要尝试类似`[119.29.29.29, 223.5.5.5]`这样的序列化写法，因为GUI会将其解析为纯字符串从而导致直接报错。

以及，个人感觉其实手搓yaml或者用大佬的yaml才是最好的，毕竟你直接从机场拿到的订阅的yaml写的通常不咋地
